binder/requirements.txt

```diff
@@ -0,0 +1,2 @@
+-r ../dev_requirements.txt
+.
```

Comment:
```suggestion
.[development]
```

This is much faster (pip resolves dependencies faster) and corresponds to the command that we use in GitHub for CI.

Reply:
Nice!